src/components/common/Input/addFarmerData.js

```diff
@@ -234,25 +234,25 @@ export const personalInfo = {
     labelName: 'Nearest Landmark',
     value: '',
     valid: false,
-    touched: false,
+    touched: false
   },
-  Phone_1: {
```

Comment:
This change will likely break the display farmer page, If you ant to change it also change it in display famrmer to lower case